setup.py

```diff
@@ -113,7 +113,7 @@ def find_version(*file_paths):
     # Ticket: 69520
     "packaging>=20.0",
     "pyparsing<3.0",
-    "pymoo==0.5.0",
+    "pymoo @ git+https://github.com/anyoptimization/pymoo.git@695cb26923903f872c7256a9013609769f3cc2bd",
```

Comment:
The reason we use a specific commit instead of tag/0.6.0 is because BootstrapNAS needs this commit: https://github.com/anyoptimization/pymoo/issues/388. This commit is now only included in the main branch.

Reply:
ok, thanks for clarification